consensus/polybft/bridge_event_relayer.go

```diff
@@ -0,0 +1,460 @@
+		}
+
+		to = ber.bridgeConfig[destinationChainID].ExternalGatewayAddr
+	}
+
+	input, err := (&contractsapi.ReceiveBatchGatewayFn{
+		Batch:     event.Batch,
+		Signature: event.Signature,
+		Bitmap:    event.Bitmap,
+	}).EncodeAbi()
+	if err != nil {
+		return err
+	}
+
+	txn := types.NewTx(types.NewLegacyTx(
+		types.WithFrom(ber.key.Address()),
+		types.WithTo(&to),
+		types.WithInput(input),
+	))
+
+	receipt, err := txRelayer.SendTransaction(txn, ber.key)
+	if err != nil {
+		return fmt.Errorf("failed to execute batch transaction on chain: %d, and gateway contract: %s. Error: %w",
+			destinationChainID, to, err)
+	}
+
+	ber.logger.Info("sent batch transaction",
+		"destinationChainID", destinationChainID,
+		"gatewayAddr", to,
+		"status", types.ReceiptStatus(receipt.Status),
+		"txHash", receipt.TransactionHash,
+		"blockNumber", receipt.BlockNumber,
+	)
+
+	return nil
+}
+
+// sendCommitValidatorSet sends commit validator set transaction to the external and internal chains
+func (ber *bridgeEventRelayerImpl) sendCommitValidatorSet(event *contractsapi.SignedValidatorSet) error {
+	input, err := (&contractsapi.CommitValidatorSetBridgeStorageFn{
+		NewValidatorSet: event.NewValidatorSet,
+		Signature:       event.Signature,
+		Bitmap:          event.Bitmap,
+	}).EncodeAbi()
+	if err != nil {
+		return err
+	}
+
+	for chainID, txRelayer := range ber.externalTxRelayers {
+		// send commit validator set transaction to the external chains
+		to := ber.bridgeConfig[chainID].ExternalGatewayAddr
+		txn := types.NewTx(types.NewLegacyTx(
+			types.WithFrom(ber.key.Address()),
+			types.WithTo(&to),
+			types.WithInput(input),
+		))
+
+		receipt, err := txRelayer.SendTransaction(txn, ber.key)
+		if err != nil {
+			// for now just log the error and continue
+			ber.logger.Error("failed to send commit validator set transaction to external chain",
+				"chainID", chainID, "error", err)
+
+			continue
+		}
+
+		ber.logger.Info("sent commit validator set transaction to external chain",
+			"chainID", chainID,
+			"gatewayAddr", to,
+			"status", types.ReceiptStatus(receipt.Status),
+			"txHash", receipt.TransactionHash,
+			"blockNumber", receipt.BlockNumber,
+		)
+
+		// send commit validator set transaction to the internal chain
+		// we create a new txn to force the getting of the nonce and estimation of gas,
+		// since the txn for external chain is modified by the external chain tx relayer
+		to = ber.bridgeConfig[chainID].InternalGatewayAddr
+		txn = types.NewTx(types.NewLegacyTx(
+			types.WithFrom(ber.key.Address()),
+			types.WithTo(&to),
+			types.WithInput(input),
+		))
+
+		receipt, err = ber.internalTxRelayer.SendTransaction(txn, ber.key)
+		if err != nil {
+			// for now just log the error and continue
+			ber.logger.Error("failed to send commit validator set transaction to internal chain", "error", err)
+
+			continue
+		}
+
+		ber.logger.Info("sent commit validator set transaction to internal chain",
```

Comment:
Lower it to debug lvl?

```suggestion
		ber.logger.Debug("sent commit validator set transaction to internal chain",
```